plugins/mumble_plugin_main.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2005-2019 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
```

Comment:
How about a file comment what this file is for (and consequently contains)?

Reply:
Sure.